lib/autocomplete-manager.coffee

```diff
@@ -98,6 +99,11 @@ class AutocompleteManager
     @subscriptions.add(atom.config.observe('autosave.enabled', (value) => @autosaveEnabled = value))
     @subscriptions.add(atom.config.observe('autocomplete-plus.backspaceTriggersAutocomplete', (value) => @backspaceTriggersAutocomplete = value))
     @subscriptions.add(atom.config.observe('autocomplete-plus.enableAutoActivation', (value) => @autoActivationEnabled = value))
+    @subscriptions.add atom.config.observe 'autocomplete-plus.suppressActivationForEditorClasses', (value) =>
+      unless value?.length
+        @suppressForClasses = []
+        return
+      @suppressForClasses = _.chain(value).reject((s) -> s.trim() is '').map((s) -> s.trim()).map((s) -> s.split('.')).value()
```

Comment:
Yep, I've picked up on that prior. This particular use of it is probably less critical than the one that occurs on each keystroke. We could likely optimize that one (`_.intersection`) away ...